service/plugin/src/main/java/com/solace/maas/ep/event/management/agent/plugin/publisher/SolacePublisher.java

```diff
@@ -27,6 +28,10 @@ public class SolacePublisher {
     private final OutboundMessageBuilder outboundMessageBuilder;
     private final DirectMessagePublisher directMessagePublisher;
 
```

Comment:
The date wasn't serializing in the command response, needed to add this module to the mapper.